index.html

```diff
@@ -144,6 +144,37 @@ <h5>LET'S CONNECT</h5>
           <hr>
         </div>
       </div>
+      <div class="card2">
+        <div class="content4">
+          <h2 class="contact-header">Contact me</h2>
+          <p class="p-contact">
+            If you have an application you are interested in developing, a 
+            feature that you need built or a project that needs coding. I’d 
+            love to help with it
+          </p>
+          <form action="">
+            <ul class="form-ul">
+              <li>
+                <label for="name"></label>
+                <input type="text" id="name"  maxlength="30" name="user_name" size="43" placeholder="Lugard"/><br><br>
+              </li>
+              <li>
+                <label for="mail"></label>
+                <input type="email" id="mail" size="43" name="user_email" placeholder="agulugard@gmail.com"/><br><br>
+              </li>
```

Comment:
- [ ] Kindly check the styling for your input as they seem to be overstretching the required width and this distorts the design. Kindly look into this.

**This is how I am seeing your design right now:**

![image](https://user-images.githubusercontent.com/86318284/194059955-cd23c036-3ecc-4a9b-8220-65ea0478fbd7.png)

**This is the required design:**

![image](https://user-images.githubusercontent.com/86318284/194060043-f755c58e-c9a8-4f5a-b7a6-46c290a56334.png)